spidermon/utils/settings.py

```diff
@@ -15,3 +16,26 @@ def getdictorlist(crawler, name, default=None):
         except ValueError:
             return value.split(",")
     return copy.deepcopy(value)
+
+
+def get_aws_credentials(settings):
+    aws_access_key_id = settings.get("SPIDERMON_AWS_ACCESS_KEY")
+    aws_secret_access_key = settings.get("SPIDERMON_AWS_SECRET_KEY")
+
+    if aws_access_key_id and aws_secret_access_key:
+        warnings.warn(
+            "SPIDERMON_AWS_ACCESS_KEY and SPIDERMON_AWS_SECRET_KEY are deprecated. "
+            "Please update them to SPIDERMON_AWS_ACCESS_KEY_ID and SPIDERMON_AWS_SECRET_ACCESS_KEY. "  # noqa
+            "Scrapy settings AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY are also valid.",
```

Comment:
Particularly, I think these messages are a bit hidden.
We should have a stat in scrapy for counting it.
Or log it as an error after a given date (the max deprecation date).
Also, what do you think of adding a max date to this one, so we are sure to come back and remove it by the time.
Otherwise, it can stay here forever...

Reply:
You mean there is no counting on deprecation warnings? We have a stat for warnings.
I don't think that this needs to have a date, we can keep this message until we decide to remove the settings from the code (maybe when we release a new major version with some backward incompatible feature).

Reply:
But it is a generic one for warnings, so it may be lost with other stuff.
This is more of concern that, we should have it somewhere, so users have an idea on when it will be removed.
This is something I see in projects like numpy, scikit-learn and pandas.
Usually they warn you with a version where it will be dropped.

Reply:
As we don't have a data for removal of this setting, I believe we can just keep this warning. When we decide to remove it, we can update this message and include the version where this settings will be dropped.